frontend-next-migration/src/preparedPages/HeroDevelopmentPage/ui/HeroDevelopmentPage.tsx

```diff
@@ -1,16 +1,189 @@
 'use client';
-import { ReactNode } from 'react';
-import { ComingSoon } from '@/widgets/ComingSoon';
+import React, { useState, useCallback, useMemo } from 'react';
+import Image from 'next/image';
+import cls from './HeroDevelopmentPage.module.scss';
+import { HeroManager, AttributesPie } from '@/entities/Hero';
+import { HeroWithGroup, HeroSlug, Stat } from '@/entities/Hero/types/hero';
+import { BarChart } from '@/entities/Hero/ui/BarChart/BarChart';
+import { color, rarityClassNames } from '@/entities/Hero/model/stats/statsDataV2';
+import { statsPricingData } from '@/entities/Hero/model/stats/statsPricingData';
+import { useClientTranslation } from '@/shared/i18n';
+import useSizes from '@/shared/lib/hooks/useSizes';
+import { classNames, Mods } from '@/shared/lib/classNames/classNames';
+import { AttributesPricing3 } from '@/entities/Hero/ui/AttributesPricing3/AttributesPricingV3';
+import HeroMenuAsDropdown from '@/features/NavigateHeroes/ui/HeroMenuAsDropdown';
```

Comment:
should be "@/features/NavigateHeroes" according to https://feature-sliced.design/